src/backend/src/transformers/vendor.transformer.ts

```diff
@@ -0,0 +1,12 @@
+import { Prisma } from '@prisma/client';
+import { Vendor } from 'shared';
+
+const vendorTransformer = (vendor: Prisma.VendorGetPayload<null>): Vendor => {
```

Comment:
you don't actually need the transformer since you're not transforming any of the data, you can remove it/ remove the map call from the service

Reply:
well we do the same thing with a regular User, I feel like for standardization we should keep it because we might want to add like a vendors projects to it at some point potentially

Reply:
good point